mozilla/fhr-jelly
# Update v4 content to run from v4 data

This updates the remote content here to be powered by the v4 client data (see [bug 1150491](https://bugzilla.mozilla.org/show_bug.cgi?id=1150491) for more background).

Simple testing steps:
* checkout
* build from top source dir: `./generate.py --output-dir html -f`
* start server in top source dir: `python -m SimpleHTTPServer 8000`
* go to `about:config` and change `datareporting.healthreport.about.reportUrl` to `http://localhost:8000/html/%LOCALE%/v4/`
* visit `about:healthreport`

Note that the v4 unit test currently is not integrated in the grunt build due to complications (old PhantomJS instance vs newer JS & no proper error output).
You can run it manually for now by navigating to `http://localhost:8000/tests/unit_v4.html`.
We are ok with running it manually for now to avoid blocking this bug, medium-term we want to switch this over to more modern tests (probably Node.js based) that have better support within Mozilla.
